### ISSUE TITLE
Release new version!!!

### ISSUE THREAD

**Issue by BataevDaniil** [erikras/redux-form#4269]

**Comment by filipposarzana** [comment]
I agree with @BataevDaniil, there are some issues fixed that have not been released yet.

**Comment by gustavohenke** [comment]
@erikras?

**Comment by davidgomes** [comment]
A new release would be very helpful to us since we depend on things that have been pushed recently but have been running our own fork to work around that.

**Comment by BataevDaniil** [comment]
The problem is solved
